app/src/sharedTest/java/org/oppia/android/app/topic/TopicFragmentTest.kt

```diff
@@ -103,6 +103,8 @@ class TopicFragmentTest {
 
   private val TOPIC_NAME = "Fractions"
 
+  private val FRACTIONS_STORY_ID = "wANbh4oOClga"
```

Comment:
Don't create a new variable for this id, import it from `StoryProgressController`

```kotlin
import org.oppia.android.domain.topic.FRACTIONS_STORY_ID_0
```

Reply:
done